### PR TITLE
NFC: fix indentation

### PR DESCRIPTION
**Pull request by brickmonster**
Fix patch indentation that has become out of sync from the surrounding code over time.
No functional changes, just less confusing.

### CODE REVIEW

File: patches/server/0325-Prevent-consuming-the-wrong-itemstack.patch [hunk 1]
```diff
@@ -28,18 +28,18 @@ index cb53fc775df387e8bbb099517793daea5ea15e4b..ba4e0bac019b68da166b277028015999
...
                      this.stopUsingItem();
--                // Paper start - if the replacement is anything but the default, update the client inventory
--                if (this instanceof ServerPlayer && !com.google.common.base.Objects.equal(defaultReplacement, itemstack)) {
-+                // Paper start
-+                if (this instanceof ServerPlayer) {
-                     ((ServerPlayer) this).getBukkitEntity().updateInventory();
-                 }
-                 // Paper end
+-                    // Paper start - if the replacement is anything but the default, update the client inventory
+-                    if (this instanceof ServerPlayer && !com.google.common.base.Objects.equal(defaultReplacement, itemstack)) {
```
Comment (brickmonster) [inline patches/server/0325-Prevent-consuming-the-wrong-itemstack.patch:40]: Is the original patch correct here? It seems to undo something intentional from an earlier patch.

Comment (brickmonster) [reply patches/server/0325-Prevent-consuming-the-wrong-itemstack.patch:40]: Patch 0072 added:
` && !com.google.common.base.Objects.equal(defaultReplacement, itemstack)`
and patch 325 reverts it.

I can fix it in a new pull request or change this request to include the fix if desired.

**Comment by brickmonster** [comment]
Rebased

**Comment by MiniDigger** [comment]
fyi, this indentation is on purpose to reduce diff

**Pull request closed without merging**